morphocut/str.py

```diff
@@ -29,11 +46,24 @@ class ParseWarning(UserWarning):
 
 @Output("meta")
 class Parse(Node):
-    """Parse information from a path.
+    """Parse information from a path. The class can be used to simply parse a string
```

Comment:
Still not path. Please read my comments carefully.

Include an explanation, maybe a quote from https://pypi.org/project/parse/

Make sure to mention:
```
:ref:`python:formatstrings`
```